### PR TITLE
Datasets for regression tests

### PR DESCRIPTION
**Pull request by matenestor**
Datasets used in tests in https://github.com/ixdat/ixdat/pull/159

### CODE REVIEW

File: failing_datasets/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+- 2024-01-17 15\_06\_41 no-ec-nonincluded -- No EC measurement ran and EC was not included in the settings dialog. Ok MS dataset.
+- 2024-01-17 15\_08\_29 no-ec-included -- No EC measurement ran, but EC was included in the settings dialog. Due to an internal bug in Zilien, it included a "pot" series filled with NaN.
```
Comment (KennethNielsen) [inline failing_datasets/README.md:2]: Maybe mention for second line that there is no count for the dataset?

File: failing_datasets/2024-01-17 15_13_45 ec-included-mpt-missing/2024-01-17 15_13_45 ec-included-mpt-missing.tsv [hunk 2]
```diff
@@ -0,0 +1,663 @@
...
+PC value_count	Number of points in the "PC value" data series	PC value	int	191
+PC setpoint_count	Number of points in the "PC setpoint" data series	PC setpoint	int	191
+C0M2_count	Number of points in the "C0M2" data series	C0M2	int	140
+C1M4_count	Number of points in the "C1M4" data series	C1M4	int	140
+C2M18_count	Number of points in the "C2M18" data series	C2M18	int	140
+C3M28_count	Number of points in the "C3M28" data series	C3M28	int	140
+C4M32_count	Number of points in the "C4M32" data series	C4M32	int	140
+C5M40_count	Number of points in the "C5M40" data series	C5M40	int	140
+C6M44_count	Number of points in the "C6M44" data series	C6M44	int	140
+pot_count	Number of points in the "pot" data series	pot	int	544
```
Comment (KennethNielsen) [inline failing_datasets/2024-01-17 15_13_45 ec-included-mpt-missing/2024-01-17 15_13_45 ec-included-mpt-missing.tsv:116]: In regards to the other PR, this is what I was referring to. Here it is simply called "pot_count". Have you got that detail under control?

File: failing_datasets/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+- 2024-01-17 15\_13\_45 ec-included-mpt-missing -- EC measurement ran, but .mpt files were not generated. Thus there is an EC-lab series header, but the series are missing.
+- 2024-01-17 15\_21\_57 ec-included -- OK EC-MS dataset.
+
```
Comment (KennethNielsen) [inline failing_datasets/README.md:5]: I think the last file is missing from the overview

**Review by KennethNielsen — CHANGES_REQUESTED** [review]
A few minor comments. Otherwise look good and great job collecting these.

File: failing_datasets/2024-01-17 15_13_45 ec-included-mpt-missing/2024-01-17 15_13_45 ec-included-mpt-missing.tsv (hunk 2 above)
Comment (matenestor) [reply failing_datasets/2024-01-17 15_13_45 ec-included-mpt-missing/2024-01-17 15_13_45 ec-included-mpt-missing.tsv:116]: I do. In datasets it is only "pot_count". Ixdat then makes it "pot_pot_count" during runtime in order to keep unique series names.
https://github.com/ixdat/ixdat/blob/e66675fc4aa81c75d6e857983e5a87434c042786/src/ixdat/readers/zilien.py#L64

**Review by KennethNielsen — APPROVED** [review]

**Review by ScottSoren — COMMENTED** [review]
Looks good!

**Pull request merged**